data/json/furniture_and_terrain/furniture-storage.json

```diff
@@ -395,7 +395,7 @@
     "id": "f_mailbox",
     "name": "mailbox",
     "symbol": "P",
-    "description": "A metal box attached to the top of a wooden post.  Mail delivery hasn't come for awhile.  Doesn't look like it's coming again anytime soon.",
+    "description": "A small metal box on top of a wooden post, designed to be a recepticle for mail deliveries.  Although considering the circumstances, it will likely never see proper use again.",
```

Comment:
```suggestion
    "description": "A small metal box on top of a wooden post, designed to receive mail deliveries.  Although considering the circumstances, it will likely never see proper use again.",
```